README.md

```diff
@@ -1,5 +1,5 @@
 <p align="left">
-    <a href="README_EN.md">English</a> ｜ 中文
+    <a href="https://github.com/OpenCSGs/csghub-sdk/blob/main/README_EN.md">English</a> ｜ 中文
```

Comment:
why use absolute path?

Reply:
pypi cannot switch readme, so link english version to github.